djangoevents/utils.py

```diff
@@ -0,0 +1,34 @@
+import inspect
+from .domain import BaseEntity, BaseAggregate, DomainEvent
+
+
+def _list_subclasses(cls):
+    """
+    Recursively lists all subclasses of `cls`.
+    """
+    return cls.__subclasses__() + [g for s in cls.__subclasses__() for g in _list_subclasses(s)]
```

Comment:
btw, interesting approach. I was mostly interested about those bits in the whole PR and it looks pretty interesting. Didn't know about `__subclasses__` method!